versioned_docs/version-0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/registering-the-worker-cluster.mdx

```diff
@@ -348,7 +351,8 @@ You must apply the values **.yaml** file on each worker cluster to install the S
 
 ## Validating the Slice Operator Installation 
 
-To validate the Slice Operator installation in the registered cluster,check the pods belonging to the namespace `kubeslice-system` by running
+To validate the Slice Operator installation in the registered cluster,check the pods belonging to the 
+namespace `kubeslice-system` by running
```

Comment:
Change namespace `kubeslice-system`  to `kubeslice-system` namespace

Reply:
Updated.